adding_species_manuscript.tex

```diff
@@ -417,11 +417,15 @@ \section*{Additions to \stdpopsim}
 which enables simulating sites with multiple mutations and possibly more than two alleles.
 Another key feature added to \stdpopsim's simulation framework was the ability to model non-crossover recombination.
 %REVISION: clarified HGT, due to rev 3 comment
-In bacteria and archaea, recombination occurs through horizontal gene transfer, which can add new genetic material from other species (e.g., via the transfer of plasmids) or replace homologous sequences in the chromosome through homologous recombination \citep{Thomas2005,Didelot2010,Gophna2022}.
-As a result, such species cannot be realistically simulated with a recombination model that considers only crossovers,
-as did the initial version of \stdpopsim.
-To address this, we made use of features of the \texttt{msprime} and \texttt{SLiM} simulation engines for modeling non-crossover homologous recombination \citep{Cury2022}.
-Modeling recombination in a bacterial or archaeal species in \stdpopsim is done by setting a flag in the species model to indicate that recombination should be modeled without crossovers,
+In bacteria and archaea, genetic material can be exchanged by through horizontal gene transfer,
+which can add new genetic material (e.g., via the transfer of plasmids)
+or replace homologous sequences through homologous recombination \citep{Thomas2005,Didelot2010,Gophna2022}.
+However, the initial version of \stdpopsim used crossover recombination to stand in for these processes.
+Although we cannot currently simulate varying gene content (as would be required to simulate horizontal gene transfer),
```

Comment:
I would suggest to be even more precise here and write
"(as would be required to simulate the addition of new genetic material by horizontal gene transfer)"
instead of
"(as would be required to simulate horizontal gene transfer)"

Background:
Some people (including me) also use the term horizontal gene transfer for homologous recombination, as it is clearly not vertical inheritance. This often causes confusion as it did for the reviewer. So in my opinion to be more precise here would not hurt.